controllers/user.controllers.js

```diff
@@ -0,0 +1,7 @@
+const { fetchUsers } = require("../models/user.models");
+
+exports.getUsers = (req, res, next) => {
+  fetchUsers().then((response) => {
+    res.status(200).send({ users: response });
+  });
```

Comment:
the connection can always fail! we should always aim to catch that potential error and tell the client the server has errored in that case!